### PR TITLE
Dependabot added

### LINKED ISSUE #661: Add Dependabot for Flutter Dependency Updates
## Description

I propose integrating Dependabot into our Flutter project to automate dependency updates. Dependabot will regularly check for updates, ensuring we stay current with the latest features and security patches. This will save time, enhance security, and maintain consistency in our development process.

### PR DESCRIPTION
**Pull request by andoriyaprashant**
## Changes

- Dependabot added
- path (.github/workflows/dependabot.yaml)

### CODE REVIEW

File: .github/workflows/dependabot.yaml [hunk 1]
```diff
@@ -0,0 +1,13 @@
+version: 2
+updates:
+  # Enable version updates for flutter's pub package manager
+  - package-ecosystem: "pub"
+    # Look for `pubspec.yaml` and `pubspec.lock` files in the `root` directory
+    directory: "/"
+    # Check the pub.dev registry for updates every day (weekdays)
+    schedule:
+      interval: "daily"
```
Comment (guyluz11) [inline .github/workflows/dependabot.yaml:9]: Can you make it monthly?

Comment (andoriyaprashant) [reply .github/workflows/dependabot.yaml:9]: yeah sure

Comment (andoriyaprashant) [reply .github/workflows/dependabot.yaml]: Done @guyluz11 👍

**Review by guyluz11 — APPROVED** [review]

**Comment by guyluz11** [comment]
Thanks for contributing :)

Consider starting the project to increase its popularity

**Pull request merged**